Point2DArr lidarCones
ConeArr cameraCones
